pyproject.toml

```diff
@@ -26,7 +26,7 @@ dependencies = [
   "jsonschema>=4.0,<5",
   "PyYAML>=6.0,<7",
   "more-itertools>=8.0,<11.0",
-  "Jinja2>=3.0,<4",
+  "Jinja2>=3.1.3,<4",
```

Comment:
If we wanted to be looser with it, we could exclude only the offending version using the [version exclusion operator](https://peps.python.org/pep-0440/#version-exclusion) which would make it look something like `Jinja2>=3.0,<4,!=3.1.2`